Description: Fix typos in code and man pages
 This patch fix some typos in code and man pages
Author: Gennaro Oliva <oliva@debian.org>
Forwarded: https://support.schedmd.com/show_bug.cgi?id=20460
Last-Update: 2024-07-21

--- a/doc/man/man5/cgroup.conf.5
+++ b/doc/man/man5/cgroup.conf.5
@@ -45,7 +45,7 @@
 \fBSystemdTimeout\fR=\fI<number>\fR
 On slow systems like virtual machines or when systemd is busy, it can take
 a lot of time to initialize and prepare the scope for slurmd during startup.
-Slurm will wait a maximum of this amount of time (in miliseconds) for the
+Slurm will wait a maximum of this amount of time (in milliseconds) for the
 scope to be ready before failing. Only applies to cgroup/v2.
 The default is 1000 ms.
 .IP
--- a/src/slurmd/slurmd/req.c
+++ b/src/slurmd/slurmd/req.c
@@ -3073,7 +3073,7 @@
 			log_flag(NODE_FEATURES, "Features on node updated successfully");
 		}
 		if (!need_reboot) {
-			log_flag(NODE_FEATURES, "Reboot not required - sending registration mesage");
+			log_flag(NODE_FEATURES, "Reboot not required - sending registration message");
 			conf->boot_time = time(NULL);
 			slurm_mutex_lock(&cached_features_mutex);
 			refresh_cached_features = true;
